### ISSUE TITLE
How to add Xref 

### ISSUE THREAD

**Issue by philip707** [AndBible/and-bible#886]
How to add existing xref to the Bible to enable reference

**Comment by tuomas2** [comment]
Are you making your own module?

**Comment by tuomas2** [comment]
There are some info about OSIS (in which you must add crossrefererencing information) here:

http://crosswire.org/osis/
https://wiki.crosswire.org/OSIS_Tutorial

**Comment by philip707** [comment]
yes... already made tamil bible and added to theword and mysword and
pocketSword

Thank you

**Comment by philip707** [comment]
> There are some info about OSIS (in which you must add crossrefererencing information) here:
> 
> http://crosswire.org/osis/
> https://wiki.crosswire.org/OSIS_Tutorial

But xref file is separate file.. just wanted to know how to include in ANDBIBLE, in other software it's included in setting folder and it automatically displayed in bible text

**Comment by tuomas2** [comment]
I'm not aware of separate xref file and And Bible does not currently support such a file, as far as I know.
Does pocketSword support it? Have you formatted the bible / xref file in OSIS format?

**Comment by philip707** [comment]
PocketSword doest support it... but mysword and theword supports same xref
files

**Comment by philip707** [comment]
but I saw a option for footnotes/xref files in ANDBIBLE

**Comment by tuomas2** [comment]
And Bible support crossreferences / footnotes in bible module, not separate files

**Comment by philip707** [comment]
Thank you..

**Comment by tuomas2** [comment]
I'm curious to know what is this file / fileformat and how PocketSword supports it. Also, do other sword apps support such a format?

**Comment by philip707** [comment]
No, PocketSword doesn't support it, but mysword and theword is supporting
it... actually I prepared this file using theword,
I am attaching a screenshot and xref in which u can see the reference bible
verse when u clik on it the verse will popup, it's using xref file,
Thank you

**Comment by philip707** [comment]
https://www.google.com/amp/s/www.thewordtutorial.com/cross-references/cross-references-introduction/amp/
If you contact Costas, developer of theword, surely he will help you to
accomadate cross reference into ANDBIBLE..

With regards

**Comment by philip707** [comment]
just now I noticed my xref file not delivered to you cause it's too large

**Comment by timbze** [comment]
Outdated
